conv/convert.go

```diff
@@ -1,16 +1,5 @@
-// Copyright 2015 go-swagger maintainers
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//    http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+// SPDX-FileCopyrightText: Copyright 2015-2025 go-swagger maintainers
```

Comment:
I wouldn't have included a -2025, because there is no legal consideration to do it I think, and now you will have to maintain the -202X every year. Because if you don't on 2026 the copyright text would be outdated, so no license would be applied.

Reply:
The year range implies active development and underline our long term commitment.
There is nothing particular to do on january 2026.
I just wanted to mark our 10th anniversary